app/controllers/reviews_controller.rb

```diff
@@ -7,4 +7,17 @@ def create
     Review.create(rating: rating, review: review_text, word_id: word, user: user)
     redirect_to word_path(params[:word_id])
   end
+
+  def edit
+    @word = Word.find(params[:word_id])
+    @review = Review.find(params[:id])
+  end
+
+  def update
+    review = Review.find(params[:id])
+    review.review = params[:review][:review]
```

Comment:
While not strictly required, your Review configuration is confusing because a "review" has a "review". I strongly encourage your to update this to "body".